web/tests/stream_list.test.js

```diff
@@ -79,7 +79,7 @@ function create_devel_sidebar_row({mock_template}) {
     stream_has_any_unread_mentions = false;
     stream_list.create_sidebar_row(devel);
     assert.equal($devel_count.text(), "42");
-    assert.equal($devel_unread_mention_info.text(), "");
+    assert.equal($devel_unread_mention_info.is(":visible"), false);
```

Comment:
@karlstolley I've update the PR to Test for the Icon's visibility right here within the test `$devel_unread_mention_info` represents the `@` icon for an unread mention